app/views/appliances/show.html.haml

```diff
@@ -61,5 +64,81 @@
           = comment.body
 
 
-.w-full.mt-8
-  = render 'appliance_comments/compact_form', appliance_comment: @new_appliance_comment
+%link{:href => "https://cdn.jsdelivr.net/npm/tailwindcss/dist/tailwind.min.css", :rel => "stylesheet"}/
+:css
+  .modal {
+    transition: opacity 0.25s ease;
+  }
+  body.modal-active {
+    overflow-x: hidden;
+    overflow-y: visible !important;
+  }
+
+
+/ Modal
+.modal.opacity-0.pointer-events-none.fixed.w-full.h-full.top-0.left-0.flex.items-center.justify-center
+  .modal-overlay.absolute.w-full.h-full.bg-gray-900.opacity-50
+  .modal-container.bg-white.md:max-w-md.mx-10.rounded.shadow-lg.z-50.overflow-y-auto{:class => "w-10/12"}
+    .modal-close.absolute.top-0.right-0.cursor-pointer.flex.flex-col.items-center.mt-4.mr-4.text-white.text-sm.z-50
+      %svg.fill-current.text-white{:height => "18", :viewbox => "0 0 18 18", :width => "18", :xmlns => "http://www.w3.org/2000/svg"}
+        %path{:d => "M14.53 4.53l-1.06-1.06L9 7.94 4.53 3.47 3.47 4.53 7.94 9l-4.47 4.47 1.06 1.06L9 10.06l4.47 4.47 1.06-1.06L10.06 9z"}
+      %span.text-sm (Esc)
+    / Add margin if you want to see some of the overlay behind the modal
+    .modal-content.py-4.text-left.px-6
+      / Title
+     
+      .title
+        %h2.text-center-3xl.font-semibold.text-gray-800.md:text-4xl
+          Segits a szintet
+          %span.text-orange-400 uptodate
+          %span.text-3xl.font-semibold.text-gray-800.md:text-4xl tartani
+        %h2.text-xl.md:text-xl.w-full
+          %strong Tolts fel egy uj jelentest 
+          %span.text-xs.font-semibold.inline-block.py-1.px-2.uppercase.rounded-full.text-orange-600.bg-orange-200.uppercase.last:mr-0.mr-1 NEW
+      
+      .form.mt-4
+        .flex.flex-col.text-sm
+        .w-full.mt-8
+          = render 'appliance_comments/compact_form', appliance_comment: @new_appliance_comment
+
+      / Footer
+      .flex.justify-end.pt-2
+        %button.modal-close.px-4.bg-blue-600.p-3.rounded-lg.text-white.hover:bg-blue-500 Close
+  :javascript
```

Comment:
The javascript code does not belong to the view. 
We could create a file `app/javascript/packs/appliance_comments/modal.js` and put the js there.
Then we can link it with.
`= javascript_pack_tag 'appliance_comments/modal', 'data-turbolinks-track': 'reload'
`